riscv/opcodes.h

```diff
@@ -1,7 +1,6 @@
 #include "encoding.h"
 
 #define ZERO	0
-#define T0      5
```

Comment:
Why delete this?

Reply:
Because I want to use `#define T0 READ_REG(X_T0)`, but `opcodes.h` will redefine it and this `T0` is not being used, so I want to remove it.

After refactoring code, I only use this once.
I will directly use `READ_REG(X_T0)` in c_sspopchk_x5.h

Reply:
My remark was that this existed in spike already and if this is a cleanup then maybe should be done as a separate PR since removing it is not related to this PR.